Assets/Scripts/UI/CameraIntMsg.cs

```diff
@@ -0,0 +1,47 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class CameraIntMsg : MonoBehaviour {
+
+    public GameObject Camera;
+    public GameObject ArrowsSide;
+    public GameObject ArrowsUp;
+    public Controller2 isghostMode;
+    private bool onlyOnce;
+
+    // Use this for initialization
+    void Start()
+    {
+        onlyOnce = false;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.C))
+        {
+
+            Debug.Log("hi");
+            Camera.SetActive(false);
+
+
+        }
+        if (Input.GetButtonDown("Horizontal") || Input.GetButtonDown("Vertical"))
+        {
+            ArrowsSide.SetActive(false);
+            ArrowsUp.SetActive(false);
+        }
+        if (isghostMode.ghostMode && !onlyOnce)
+        {
+            ArrowsUp.SetActive(false);
+            ArrowsSide.SetActive(true);
+            onlyOnce = true;
+        }
+        if (Input.GetKeyDown(KeyCode.Z))
+        {
+
+            ArrowsSide.SetActive(false);
```

Comment:
Making the whole prefab as a doll is much easier...all you gotta do is drag and drop the doll. I will make it so that you dont have to drag and drop stuff :+1: